Framework/Geometry/test/ComponentHelperTest.h

```diff
@@ -1,393 +0,0 @@
-#ifndef MANTID_GEOMETRY_COMPONENTHELPERTEST_H_
```

Comment:
Because I removed `moveComponent` and `rotateComponent` and there were no tests for the other functions that remain.